layers/+tools/fzf/config.vim

```diff
@@ -23,5 +23,56 @@ else
   nnoremap <Leader>f? :Files<CR>
 
   nnoremap <Leader>ff :Files ~<CR>
+
+      " fzf
+  "fzf work with mhinz/vim-startify on sessions
+  function! s:sessions()
+    call fzf#run({
+    \ 'source':  'ls -1 ~/.vim/session',
+    \ 'sink':    'SLoad',
+    \ 'options': '+m --prompt="Sessions> "',
+    \ 'down':    '40%'
+    \})
+  endfunction
+  command! Sessions call s:sessions()
+  nnoremap <silent> <leader>sf :Sessions<cr>
```

Comment:
Move this block, `s:sessions()` to `core/autoload/spacevim/plug/fzf.vim`, `spacevim#plug#fzf#Session()`.

```vim
nnoremap <silent> <Leader>fs :call spacevim#plug#fzf#Session()<CR>
```